examples/neg/namer/issue950/effect_operation_with_same_name.effekt

```diff
@@ -0,0 +1,11 @@
+interface Greet { def sayHello(): Unit }
```

Comment:
Please also add one more negtest an overloaded interface: two methods of different interfaces of the same name, just to see what it does for that situation... (to see if it's actually testing the `foreach` loop or not)